server/src/test/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStoreTests.java

```diff
@@ -206,32 +210,46 @@ void retrieveNonExistentToken() {
 
     @Test
     void cleanUpExpiredTokensBasedOnExpiresField() {
+        TestClock clock = new TestClock();
+        Duration expirationTime = Duration.ofHours(1);
+        store = UaaTokenStore.constructorForTesting(dataSource, expirationTime, clock);
+
         int count = 10;
         String lastCode = null;
         for (int i = 0; i < count; i++) {
+            clock.instants.add(Instant.now());
             lastCode = store.createAuthorizationCode(clientAuthentication);
         }
         assertThat(jdbcTemplate.queryForObject("SELECT count(*) FROM oauth_code", Integer.class), is(count));
 
         jdbcTemplate.update("UPDATE oauth_code SET expiresat = ?", System.currentTimeMillis() - 60000);
 
         final String finalLastCode = lastCode;
+        clock.instants.add(Instant.now().plus(expirationTime));
+        clock.instants.add(Instant.now().plus(expirationTime));
+
         assertThrows(InvalidGrantException.class, () -> store.consumeAuthorizationCode(finalLastCode));
         assertThat(jdbcTemplate.queryForObject("SELECT count(*) FROM oauth_code", Integer.class), is(0));
     }
 
     @Test
     void cleanUpLegacyCodesCodesWithoutExpiresAtAfter3Days() {
```

Comment:
This test method name is unparseable. Could it be reduced to English?

Reply:
I don't disagree. I hadn't read it until now.

It's a bit awkward to read a comment about an old line that I didn't touch. Are you asking me to fix that in this PR?

Reply:
Yes, though I wouldn't insist on it since you didn't introduce that name in this PR.

Reply:
OK the test was indeed nasty. I refactored it, and renamed the test.